custom_commands/suppressedNotificationsTrue.js

```diff
@@ -0,0 +1,50 @@
+const axios = require('axios');
+
+module.exports = {
+  command: async function () {
+    
+
+
+const getToken = async () => {
+    const response = await axios.post(
+        'https://qa2.development.tidepool.org/auth/login',
+        '',
+        {
+            headers: {
+                'Authorization': 'Basic YnJpYW4rdXBsb2FkNzg2d2ViMTMyN2NsaW5pYzM1YUB0aWRlcG9vbC5vcmc6dGlkZXBvb2w=',
```

Comment:
This is not a token, those are base64 encoded credentials - consider rotating the password.

Reply:
Yup, I wrote it wrong 🤝 We can generated it from credentials saved in env and it should be enough, we don't need to rotate passwords IMHO

Reply:
This particular password has now been exposed to the entire world, given how this is a public repo. It may already be in https://archiveprogram.github.com/arctic-vault/ 😬

Reply:
i've changed the password for this account just now. ill look into rotation.